models/db/achievement.d.ts

```diff
@@ -0,0 +1,13 @@
+import { Types } from 'mongoose';
+import AchievementType from '../../constants/achievementType';
+import { EnrichedLevel } from './level';
+import User from './user';
+
+interface Achievement {
+    type: AchievementType;
+    tag: string;
```

Comment:
what is tag for? I think we should have a map somewhere of AchievementType to description strings, and then we can just reference the map when we need to write it

Reply:
There was a comment @k2xl had made that mentioned allowing a basic data type to hold the achievement type but I believe you guys have sold that issue above?

Reply:
Type is already stored in `type` so should be fine to remove `tag`. `type` will be referenced in the code elsewhere to convert to the corresponding string